### PR TITLE
supercharged-dhbw:3.0.0

### PR DESCRIPTION
**Pull request by DannySeidel**
I am submitting
- [ ] a new package
- [x] an update for a package

**Description:**
- combine header settings in a dictionary (breaking change)
- move abstract in front of the outlines
- refactor language handling (thanks to @B1TC0R3)
- smaller improvements to the code readability (thanks to @B1TC0R3)
- provide more examples (see https://github.com/DannySeidel/typst-dhbw-template/pull/11)
- attribute to provide a custom declaration of authorship (see https://github.com/DannySeidel/typst-dhbw-template/pull/11)

I have read and followed the submission guidelines and, in particular, I
- [x] selected a name that isn't the most obvious or canonical name for what the package does
- [x] added a `typst.toml` file with all required keys
- [x] added a `README.md` with documentation for my package
- [x] have chosen a license and added a `LICENSE` file or linked one in my `README.md`
- [x] tested my package locally on my system and it worked
- [x] `exclude`d PDFs or README images, if any, but not the LICENSE

- [x] ensured that my package is licensed such that users can use and distribute the contents of its template directory without restriction, after modifying them through normal use.

### CODE REVIEW

**Comment by B1TC0R3** [comment]
If it works fine, then all is good. :+1:

File: packages/preview/supercharged-dhbw/3.0.0/template/main.typ [hunk 1]
```diff
@@ -0,0 +1,130 @@
+#import "@local/supercharged-dhbw:3.0.0": *
```
Comment (elegaanz) [inline packages/preview/supercharged-dhbw/3.0.0/template/main.typ:1]: Can you please update this import to use the `@preview` namespace?

**Review by elegaanz — COMMENTED** [review]
Just one thing, and it should be good to merge.

File: packages/preview/supercharged-dhbw/3.0.0/template/main.typ (hunk 1 above)
Comment (DannySeidel) [reply packages/preview/supercharged-dhbw/3.0.0/template/main.typ]: Thanks for pointing that out 🙏🏻

I forgot to change it back 🙂

**Comment by elegaanz** [comment]
Thanks :)

**Pull request merged**